content/workshops/openshift33/exercise1.3.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Exercise 1.3 - Developing and Managing Your Application
+workshops: openshift33
+workshop_weight: 130
+layout: lab
+---
+
+### Developing and managing an application in OpenShift
+In this lab we will explore some of the common activities undertaken by developers working in OpenShift.  You will become familiar with how to use environment variables, secrets, build configurations, and more.  Let's look at some of the basic things a developer might care about for a deployed app.
+
+### Setup
+From the previous lab you should have the DC Metro Maps web app running in OpenShift.
+
+<!-- :warning: **Only if you don't already have it running already, add it with the following steps.** -->
```

Comment:
Delete this line instead of commenting it out.

Find any other places where this is done and delete them there too.